spec/mixins/webhook_spec.rb

```diff
@@ -1,29 +1,38 @@
 require "spec_helper"
 
-RSpec.describe Rubyblok::Mixins::Webhook, type: :controller do
-  describe "#create" do
-
-    let(:subject) { Class.new { include Rubyblok::Mixins::Webhook } }
-    let(:instance) { subject.new }
+class Page
```

Comment:
What if we define these methods on Page? In that case we don't need to mock it.
```
def self.find_or_initialize_by(storyblok_story_id:)
  Page.new
end

def update(attributes)
end
```

Reply:
Didn't worked, I pushed the latest changes in this one.